notebooks/simple-xgboost/README.md

```diff
@@ -0,0 +1,50 @@
+# Trtion FIL backend with XGBoost
+
+This notebook is a reference for deploying a forest model trained using XGBoost library in Triton Inference Server with Forest Inference Library (FIL) backend. The notebook explains how one can deploy XGBoost model in Triton, check deployment status and send inference requests, set concurrent model execution and dynamic batching and find the best deployment configuration using Model Analyzer.
```

Comment:
Seems a bit odd to have the acronyms expanded only here and nowhere else

Reply:
Agreed. I'm trying not to make too significant of changes to the stuff that was here already, but that seems worth a touchup. I'll take care of it.